gui-tk/csvImport.tcl

```diff
@@ -138,7 +138,7 @@ proc csvImportDialogOK {url} {
         cancelWin .wiring.csvImport
         # Delete temporary file created by web import of CSV data
         if [string length $url] {
-	        file delete $filename      
+	        csvDialog.deleteFile $filename   
```

Comment:
This was not what I had in mind. There is no benefit to providing a C++ implementation of a file deleter, when TCL already provides one.

What I had in mind was that you provide a variant of MinskyTCL::loadVariableFromCSV that takes a URL, and manages any temporary file it might create within it.

You can probably even do away with storing anything to disk, as boost:beast probably provides a streaming interface.

Reply:
Sorry, I misunderstood. I thought that you didn't want a file delete method in TCL because it was more unreliable than C++, an impression that was reinforced by my own experience that files would sometimes not be deleted during web import. I will implement it the way you suggest, thanks.